CHANGELOG.rst

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 ------------------
 
 - Fix: prevent caching large objects on fill_cache [lferran]
+- Fix: patch a role on dbusers:groups don't remove users from them
```

Comment:
Could u add your username?

Reply:
and move your lines above the existing ones :P (so it matches the git history)